i18n/generate.py

```diff
@@ -135,19 +139,18 @@ def clean_pofile(pofile_path):
     return duplicate_entries
 
 
-def validate_files(directory, files_to_merge):
+def validate_file(directory, filename):
     """
     Asserts that the given files exist.
     files_to_merge is a list of file names (no directories).
     directory is the directory (a path object from path.py) in which the files should appear.
     raises an Exception if any of the files are not in dir.
     """
-    for file_path in files_to_merge:
-        pathname = directory.joinpath(file_path)
-        if not pathname.exists():
-            raise Exception(f"I18N: Cannot generate because file not found: {pathname}")
-        # clean sources
-        clean_pofile(pathname)
+    pathname = directory.joinpath(filename)
+    if not pathname.exists():
+        raise ValueError(u"I18N: Cannot generate because file not found: {0}".format(pathname))
```

Comment:
Do we need this unicode `u" ... "` as we have migrated to python3+?

Reply:
No we don't -- this comes from a difficult rebase.